GDevelop.js/Bindings/Bindings.idl

```diff
@@ -654,6 +654,23 @@ interface Effect {
   void UnserializeFrom([Const, Ref] SerializerElement element);
 };
 
+interface EffectsContainer {
```

Comment:
Great :) I suggest you add also some basic tests in `GDevelop.js/__tests__/Core.js`.
This will allow you to try this without running the whole IDE, and be confident that it works well :) Basically, create tests that make an effects container, add a few effects, check it can get them back, maybe serialize/unserialize.